builder/vsphere/driver/driver_test.go

```diff
@@ -37,3 +49,59 @@ func newVMName() string {
 	rand.Seed(time.Now().UTC().UnixNano())
 	return fmt.Sprintf("test-%v", rand.Intn(1000))
 }
+
+func NewSimulatorServer(model *simulator.Model) (*simulator.Server, error) {
+	err := model.Create()
+	if err != nil {
+		return nil, err
+	}
+
+	model.Service.RegisterEndpoints = true
+	model.Service.TLS = new(tls.Config)
+	model.Service.ServeMux = http.NewServeMux()
+	return model.Service.NewServer(), nil
+}
+
+func NewSimulatorDriver(s *simulator.Server) (Driver, error) {
+	ctx := context.TODO()
+
+	credentials := url.UserPassword("user", "pass")
```

Comment:
do you think it would be a good idea to allow passing specific user/pass here, I'm asking this because I saw some urlencoding issue for password around there.

Reply:
Is doesn't make a difference to change these credentials, it can be any value. I'll remove the credentials cuz right now is not explicit that can be anything.  I just need an `url.UserInfo` object here

Reply:
gotcha !